force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +447,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.getAge() > person2.getAge()){
+            return person1;
+        }else if (person1.getAge() < person2.getAge()){
+            return person2;
+        }else{
+        return person1;
+        } // Replace null with the variable you used to store the result
```

Comment:
Good but probably one more conditional branch than you really need. If person1 is >= to person2 then return person1. Else return person2